src/main/kotlin/no/nav/syfo/behandler/database/BehandlerExtendedQuery.kt

```diff
@@ -46,7 +46,7 @@ SELECT b.id behandlerid, b.her_id behandlerherid, b.created_at behandlercreateda
         INNER JOIN BEHANDLER_KONTOR AS k ON (k.id = b.kontor_id)
         WHERE k.dialogmelding_enabled IS NOT NULL 
         AND (b.fornavn ilike ? OR b.etternavn ilike ? 
-            OR position(? IN k.navn)>0 OR position(UPPER(?) IN k.navn)>0 
+            OR position(? IN k.navn)>0 OR position(UPPER(?) IN k.navn)>0 OR position(INITCAP(?) IN k.navn)>0 
```

Comment:
Er det ikke enklere å gjøre søkestrengen uppercase?

Reply:
Ja, man kan kunne brukt 
position(UPPER(?) IN UPPER(k.navn))>0
men jeg tror det blir tungt for databasen siden den da i prinsippet må lage en slags kopi av hele tabellen. Dessverre finnes det ikke noen case-insensitiv variant av position.